nav2_planner/src/planner_server.cpp

```diff
@@ -510,7 +517,72 @@ PlannerServer::publishPlan(const nav_msgs::msg::Path & path)
   }
 }
 
-}  // namespace nav2_planner
+void PlannerServer::isPathValid(
+  const std::shared_ptr<nav2_msgs::srv::IsPathValid::Request> request,
+  std::shared_ptr<nav2_msgs::srv::IsPathValid::Response> response)
+{
+  unsigned int mx = 0;
+  unsigned int my = 0;
+
+  response->is_valid = true;
+
+  if (request->path.poses.empty() ) {
+    response->is_valid = false;
```

Comment:
you can add a `return` here and then the rest of this function doesn't need to be in an `else` statement, which makes it easier to read

Reply:
Done